public/index.html

```diff
@@ -13,8 +13,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
It looks like this was used with IE, but IE is retired. The index.html that ships with Vite doesn't include this tag. I've seen recommendations not to include this tag anymore.